code/game/gamemodes/changeling/changeling_powers.dm

```diff
@@ -967,3 +967,37 @@ var/list/datum/dna/hivemind_bank = list()
 	feedback_add_details("changeling_powers","ED")
 	return 1
 
+/mob/proc/changeling_armblade()
+	set category = "Changeling"
+	set name = "Generate Arm Blade (20)"
+	set desc="Transform one of our arms into a deadly blade."
+
+	if(!istype(src, /mob/living/carbon/human)) return
```

Comment:
It's not exactly a port.
Also I advised him to use the `/mob/proc/` path because all the other changeling powers are `/mob/proc/` and I didn't want to break anything.
I advised him to restrict it to humans because `can_use_hand()` was needed to check whether the changeling had a hand and `can_use_hand()` is a human-level proc.